driver/driver.go

```diff
@@ -118,7 +132,46 @@ func NewDriver(url, endpoint, nodeID, account string, mounter *mount.SafeFormatA
 		nodeCaps: []csi.NodeServiceCapability_RPC_Type{
 			csi.NodeServiceCapability_RPC_STAGE_UNSTAGE_VOLUME,
 		},
-	}, nil
+		attacher: attacher,
+		quit:     make(chan bool),
+	}
+
+	driver.log.Info("Starting JWT maintainer to refresh the JWT at least once each 30 days.")
+	driver.client.JWT.Get()
+	driver.jwtRefresher = time.NewTicker(29 * 24 * time.Hour)
```

Comment:
`jwtRefresher` doesn't have to be a member of `Driver`, it's only used inside the goroutine below.